docs/README.md

```diff
@@ -14,7 +14,7 @@ Install [mkdocs](https://www.mkdocs.org/#installation).
 
 ## Test and deploy locally
 
-To test out changes locally, run:
+To test out changes locally, run from /runscanner:
```

Comment:
"from the repository root directory" might be more accurate as it doesn't have to be named `runscanner`

Reply:
Fixed